V3SampleApp/MasterViewController.swift

```diff
@@ -8,19 +8,33 @@
 
 import UIKit
 import StreamChatClient_v3
+import Combine
 
 class MasterViewController: UITableViewController {
 
     lazy var channelListController: ChannelListController = chatClient
         .channelListController(query: ChannelListQuery(filter: .in("members", ["broken-waterfall-5"]), options: [.watch]))
-    
+
+    @available(iOS 13 ,*)
+    lazy var channelListControllerPublishers: ChannelListController.Publishers = chatClient
+        .channelListControllerPublishers(query: ChannelListQuery(filter: .in("members", ["broken-waterfall-5"]), options: [.watch]))
```

Comment:
I think should be easer way to get publishers if you already have `channelListController`.
Something like this: `channelListController.publishers` and use this API in Client extension.